app.js

```diff
@@ -57,7 +57,7 @@ const path = require('path');
 // aren't
 
 try {
-	require('sugar');
+	require.resolve('sugar-deprecated');
```

Comment:
That wouldn't automatically install new dependencies. I have always seen these instructions as intended to making sure that all the dependencies are installed.